voctopublish/api_client/voctoweb_client.py

```diff
@@ -368,7 +376,11 @@ def create_recording(self, local_filename, filename, folder, language, hq, html5
         try:
             # todo ssl verify by config
             # r = requests.post(url, headers=headers, data=json.dumps(payload), verify=False)
-            r = requests.post(url, headers=headers, data=json.dumps(payload))
+            if recording_id:
+                r = requests.patch(url, headers=headers, data=json.dumps(payload))
```

Comment:
I would move (and duplicate) the logging inside this if/else, so that the logging includes the HTTP method.

Reply:
Wäre für mich ein eigener PR, das Logging vorher ja auch schon nicht da war...